source/options.html

```diff
@@ -1,8 +1,6 @@
 <!doctype html>
 <meta charset="utf-8">
 <title>Options</title>
-<link rel="stylesheet" href="chrome://global/skin/in-content/common.css">
```

Comment:
Required for Firefox

Reply:
webext-base-css already includes it and it seems to work correctly in FF: https://github.com/fregante/webext-base-css/blob/main/webext-base.css#L7